create_student.rb

```diff
@@ -18,5 +20,6 @@ def create_student
 
   student = Student.new(age, nil, name, parent_permission: permission)
   @people.push(student)
+  save_student(age, name, permission)
   puts 'A student is created successfully'
```

Comment:
- [ ] when creating a student the user receives below error message:

![afbeelding](https://user-images.githubusercontent.com/39240820/218078077-f08f34b6-e5e5-4bbd-bd2a-71e75a18661b.png)